it/testSupport/preConditions/SdilBackendStub.scala

```diff
@@ -192,6 +192,15 @@ case class SdilBackendStub()
     builder
   }
 
+  def submitReturnsVariation(sdilRef: String)= {
+    stubFor(
+      post(
+        urlPathMatching(s"/returns/vary/$sdilRef"))
+        .willReturn(
+          noContent()))
+    builder
+  }
+
   def submitVariationError(sdilRef: String) = {
     stubFor(
       post(
```

Comment:
What about the possibility of changed sdil activity - i.e. with the new copacker or new importer mini journey?  Where is that being handled/sent?

Reply:
![image](https://github.com/hmrc/soft-drinks-industry-levy-variations-frontend/assets/75027964/a66eeed6-69f1-4860-91cb-144120b7ced1)

Reply:
When checking through the old service, even when changing the subscription to a new importer and adding a new warehouse this isn't submitted within the return adjustment